src/modules/logger/logger.cpp

```diff
@@ -1399,11 +1392,11 @@ void Logger::write_load_output()
 	callback_data.logger = this;
 	callback_data.counter = 0;
 	callback_data.buffer = buffer;
-	hrt_abstime curr_time = hrt_absolute_time();
 	_writer.set_need_reliable_transfer(true);
 	// TODO: maybe we should restrict the output to a selected backend (eg. when file logging is running
 	// and mavlink log is started, this will be added to the file as well)
-	print_load_buffer(curr_time, buffer, sizeof(buffer), print_load_callback, &callback_data, &_load);
+	print_load_buffer(buffer, sizeof(buffer), print_load_callback, &callback_data, &_load);
+	cpuload_monitor_stop();
```

Comment:
What if `top` is running  at the same time?

Reply:
Then only idle will continue to update. I glossed over that case as something I didn't think was a real issue, but we could add a simple count instead if you think anyone might actually hit it.

Reply:
Yes I'd use a counter instead of the bool.

Reply:
Done.